LayoutTests/http/tests/performance/performance-resource-timing-entries-default-limit.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<script src="../../resources/js-test-pre.js"></script>
+<script>
+
+for (let i = 0; i < 300; ++i)
+    document.write(`<img src="../../resources/square100.png?${i}">`);
+
+window.addEventListener("load", function() {
+    shouldBe('performance.getEntriesByType("resource").length', '250');
+});
+</script>
+<script src="../../resources/js-test-post.js"></script>
```

Comment:
Not needed.

Reply:
well, if we could include js-test.js then yes but we can't so I'd keep it around.

Reply:
You can, just grep'ing `js-test.js` in http/tests will show you many examples.